src/utils/normalizeToNumberString/__tests__/normalizeToNumberString.test.ts

```diff
@@ -15,6 +15,20 @@ describe('normalizeToNumberString', () => {
 
         expect(normalizedNumberString).toBe('22.22');
     });
+    it('should give a correct output integer value without prefix', () => {
```

Comment:
add tests for prefix and post fix.
add test for negative numbers. "-" sign preceded by a number is will be dropped